.github/workflows/preview.yml

```diff
@@ -0,0 +1,37 @@
+name: Storybook preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+      - edited
```

Comment:
Why do we need to redeploy when PR info is edited? Here the author of the GH action mentions that we should run the action only when files change: https://github.com/rossjrw/pr-preview-action#run-only-when-files-are-changed

Reply:
This is for trigger the deployment. For my testing. Should be removed before merging. I added it ONLY for triggering the deployment. I should have had `revert` this but it was late and I didn't think about it. You read correctly

Reply:
So, if we do changes in this PR the workflow won't trigger without this thing.

Reply:
Added comment here https://github.com/warp-ds/react/pull/136/files#r1368234462